dense_vector/challenges/default.json

```diff
@@ -23,10 +23,52 @@
       }
     },
     {
-      "operation": "index-append",
+      "name": "index-append",
+      "operation": {
+        "operation-type": "bulk",
+        "bulk-size": {{bulk_size | default(5000)}},
+        "ingest-percentage": 10
+      },
       "warmup-time-period": {{ bulk_warmup | default(40) | int }},
       "clients": {{bulk_indexing_clients | default(1)}}
     },
+    {
+      "parallel": {
```

Comment:
without a completed-by here, the block will execute until all dependent tasks have completed. Note however, that if the indexing task completes before the querying tasks (I'm not sure how likely this is), you will have querying tasks that execute and affect your results but are not actually executing in parallel to indexing